src/ORMSetup.php

```diff
@@ -34,7 +34,7 @@ public static function createAttributeMetadataConfiguration(
         CacheItemPoolInterface|null $cache = null,
     ): Configuration {
         $config = self::createConfiguration($isDevMode, $proxyDir, $cache);
-        $config->setMetadataDriverImpl(new AttributeDriver($paths));
+        $config->setMetadataDriverImpl(new AttributeDriver($paths, true));
```

Comment:
Why are you touching this?

Reply:
Oh, I overlooked that, in previous implementation I had 3. parameter in AttributeDriver and when I refactored it I just deleted and this is what was left.